tests/Aspire.Hosting.Oracle.Tests/Aspire.Hosting.Oracle.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Aspire.Hosting.AppHost\Aspire.Hosting.AppHost.csproj" />
+    <ProjectReference Include="..\Aspire.Hosting.Tests\Aspire.Hosting.Tests.csproj" />
+    <ProjectReference Include="..\..\src\Aspire.Hosting.Oracle\Aspire.Hosting.Oracle.csproj" />
+    <ProjectReference Include="..\..\src\Components\Aspire.Oracle.EntityFrameworkCore\Aspire.Oracle.EntityFrameworkCore.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="$(RepoRoot)src\Aspire.Hosting.Oracle\OracleContainerImageTags.cs" />
+    <Compile Include="$(SharedDir)VolumeNameGenerator.cs" Link="Utils\VolumeNameGenerator.cs" />
+    <Compile Include="$(RepoRoot)tests\Aspire.Hosting.Tests\Utils\LoggerNotificationExtensions.cs" Link="Utils\LoggerNotificationExtensions.cs" />
```

Comment:
Why don't we just make LoggerNotificationExtensions `public` in Aspire.Hosting.Tests, and then Aspire.Hosting.Oracle can use it normally? That way we don't have to shared source compile it into multiple assemblies.

Reply:
If I do that will you ask that I create an API Design before it gets merged?

Reply:
I don't think we need an API design for APIs in our tests.